course02/homework02/Tomasz/message_throttler_interface.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+template<
+	typename _Message,
+	typename _MessageConsumer,
+	typename _MessageDisposer,
+	typename _Timestamp,
+	typename _Timestamper,
+	typename _TimestampThreshold,
+	typename _Buffer
+>
+class message_throttler_interface
```

Comment:
Why calling it interface?

Reply:
It was the best name that came to my mind as it gives access to the throttler for a single user.